petvax-app/petvax-app/src/app/components/header/header.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="container" >
+<div style= "float: left;"><img src="../../../assets/images/logo.png" height="150" width="350"/></div>
```

Comment:
Inline CSS is useful for brief properties that are not expected to be reused. Since this div only has a float property and the image only adjusts height and width, I think this complies with this purpose. Also, these properties are not expected to be reused in order to assign a "class" and add it to the CSS.